### ISSUE TITLE
Renaming package?

### ISSUE THREAD

**Issue by lust4life** [easysql/easy_sql#23]
Shall we consider renaming the package name? As the current name : [easy-sql-easy-sql ](https://pypi.org/project/easy-sql-easy-sql/) is a little bit verbose and hard to give it a quick install. 

proposals: `esql`, e can be interpreted as _easy, enhancement, etl, enlighten_ .... 😜  😸

**Comment by gmlove** [comment]
It's because there is another easy-sql pip package there.
It's worth considering but I'm concerned about that the related change is massive and easy to miss things. Anyway, user will only need to care about the name during installation. For the usage in code, it's fine. I personally would like to postpone it for now.

**Comment by lust4life** [comment]
Yeah, agree. adapting the whole code base might not be worth trying at the moment, maybe only change the package name in PYPI ? But it will bring us the gap between the package name and the module name used in the code. 🤔
